source/public/styles/styles2.css

```diff
@@ -0,0 +1,415 @@
+
+/*Styling for Home Page!*/
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap');
+
+*, *::before, *::after{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0; 
+    font-family: 'Inter', "FontAwesome", sans-serif;
+    
+}
+
+#homepage {
+    height:15%;
+    width:85%; /*Percentage based on parent container size*/
+    margin: 0 auto; /*Sets top and bottom margin to 0. but horizontal auto adjusts according to browser*/
+    display:flex;
+    align-items: center; /*Aligns content vertically in the center of the container*/
+    justify-content: center; /*Aligns flex items horizontally*/
+}
+
+#homepage h1{
+    color: #FF9B83;
+}
+
+main{
+    display:flex;
+    flex-grow: 1;
+    justify-content: space-evenly;
+}
+
```

Comment:
Small thing for the buttons on the page, could you add some sort of visual indication like a cursor change on hover to indicate that it's clickable, right now it's kinda hard to tell